src/routes/+page.svelte

```diff
@@ -65,8 +61,13 @@
 			sourceLayer="rnet_limerick"
 			hoverCursor="pointer"
 		>
-			<Popup openOn="click" offset={[0, -10]}>
-				<div class="text-lg font-bold">"Test value"</div>
+			<Popup openOn="click" offset={[0, -10]} let:features>
+				{@const props = features?.[0]?.properties}
```

Comment:
This is the magic I need to get my head around. One line of code, lots to digest!

Reply:
not sure my summary will be fully correct, but can try;  The component exposes some of it's state. to hook into that use `let:[theVarNameExposed]` now that is available for you to use within the children of the component that you define up here.